frontend/src/lib/components/MultiSelect.svelte

```diff
@@ -9,7 +9,7 @@
 
 <MultiSelect
 	bind:selected
-	{options}
+	options={options.toSorted()}
```

Comment:
what does this achieve?

Reply:
when e-mails are sorted alphabetically, then users can easily find the emails they are looking for

Reply:
good touch, approved